tests/end_to_end/renew_certificate.py

```diff
@@ -135,7 +135,7 @@ def test_run_configuration(self):
         CertificateTest(self.test).step_cert_import()
         CertificateTest(self.test).step_cert_register()
         CertificateTest(self.test).step_cert_activate()
-
+        time.sleep(460)
```

Comment:
time.sleep is not a good method to check whether some operation has finished, please check for example waitfor() in tests like CertificateTest().